monitoring/cluster_observability_operator/cluster-observability-operator-release-notes.adoc

```diff
@@ -16,6 +16,16 @@ The {coo-short} complements the built-in monitoring capabilities of {product-tit
 
 These release notes track the development of the {coo-full} in {product-title}.
 
+[id="cluster-observability-operator-release-notes-0-1-3"]
+== {coo-full} 0.1.3
+{coo-full} 0.1.3 is a bug fix release.
```

Comment:
sorry, I am not sure if it's accurate using  - "a bug fix release" or "not a bug fix release". I am wondering if this can be reworded? 
Maybe "{coo-full} 0.1.3 contains a bug fix."

Reply:
Good catch. I will add the advisory link.